NEWS.txt

```diff
@@ -3,182 +3,61 @@ Release Notes
-
-A total of 65 people contributed to this release since 0.9.0, see list below.
-
-Interface changes:
- - Merged duplicated diagonal functions into two ops: ``ExtractDiag`` (extract a diagonal to a vector),
-   and ``AllocDiag`` (set a vector as a diagonal of an empty array)
- - Renamed ``MultinomialWOReplacementFromUniform`` to ``ChoiceFromUniform``
-
- - Removed or deprecated Theano flags:
-
-   - ``cublas.lib``
-   - ``cuda.enabled``
-   - ``enable_initial_driver_test``
-   - ``gpuarray.sync``
-   - ``home``
-   - ``lib.cnmem``
-   - ``nvcc.*`` flags
-   - ``pycuda.init``
-
- - Changed ``grad()`` method to ``L_op()`` in ops that need the outputs to compute gradient
+A total of 14 people contributed to this release since 0.10.0beta1, see list below.
 
 Convolution updates:
- - Extended Theano flag ``dnn.enabled`` with new option ``no_check`` to help speed up cuDNN importation
- - Implemented separable convolutions
- - Implemented grouped convolutions
+ - Added 3D separable convolutions
+ - Added 3D grouped convolutions
+ - Removed old ``conv3d`` interface
+ - Deprecated old ``conv2d`` interface
+ - Updated ``conv`` documentation
 
 GPU:
- - Prevent GPU initialization when not required
- - Added disk caching option for kernels
- - Added method ``my_theano_function.sync_shared()`` to help synchronize GPU Theano functions
- - Added useful stats for GPU in profile mode
- - Added Cholesky op based on ``cusolver`` backend
- - Added GPU ops based on `magma library <http://icl.cs.utk.edu/magma/software/>`_:
-   SVD, matrix inverse, QR, cholesky and eigh
- - Added ``GpuCublasTriangularSolve``
- - Added atomic addition and exchange for ``long long`` values in ``GpuAdvancedIncSubtensor1_dev20``
- - Support log gamma function for all non-complex types
- - Support GPU SoftMax in both OpenCL and CUDA
- - Support offset parameter ``k`` for ``GpuEye``
- - ``CrossentropyCategorical1Hot`` and its gradient are now lifted to GPU
-
- - Better cuDNN support
+ - Added a meta-optimizer to select the fastest GPU implementations for convolutions
 
-   - Official support for ``v5.*`` and ``v6.*``
-   - Better support and loading on Windows and Mac
-   - Support cuDNN v6 dilated convolutions
-   - Support cuDNN v6 reductions
-   - Added new Theano flags ``cuda.include_path``, ``dnn.base_path`` and ``dnn.bin_path``
-     to help configure Theano when CUDA and cuDNN can not be found automatically.
+ - cuDNN:
 
- - Updated ``float16`` support
-
-   - Added documentation for GPU float16 ops
-   - Support ``float16`` for ``GpuGemmBatch``
-   - Started to use ``float32`` precision for computations that don't support ``float16`` on GPU
+   - Official support for ``v6.*`` and ``v7.*``, support for ``v5.*`` will be removed in next release
+   - Added spatial transformation operation based on cuDNN
+   - Updated and improved caching system for runtime-chosen cuDNN convolution algorithms
+   - Support cuDNN v7 tensor core operations for convolutions
```

Comment:
Indeed. And currently, it is even only for time_(once|on_shape_change) algorithms.